modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java

```diff
@@ -87,7 +87,7 @@ class Item {
           .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Not part of this patch, but since it showed up in the diff, I'll comment anyway. :stuck_out_tongue: 

See similar line below.